packages/stark-ui/src/modules/slider/components/slider.component.ts

```diff
@@ -110,6 +110,13 @@ export class StarkSliderComponent extends AbstractStarkUiComponent implements Af
 	 * Component lifecycle hook that is called after data-bound properties of a directive are initialized.
 	 */
 	public ngOnInit(): void {
+		if (!this.values) {
+			throw new Error("property values should be set on stark-slider");
```

Comment:
Can you add a unit test for this? And also can you adapt the error message to align with the convention we follow for other errors: `StarkXXDirective: error message.`?